### DISCUSSION TITLE
Noah-MP-crop模块模拟小麦

### DISCUSSION THREAD

**Discussion by 12wlj** [NCAR/hrldas#198] [Q&A]
您好，请问Noah-MP-crop模块来模拟小麦，小麦的生长参数有没有可以参考的范围。如果有参考范围，可以分享一下吗。

**Comment by cenlinhe** [comment]
@CharlesZheZhang

**Comment by CharlesZheZhang** [comment]
Thank you for your question, please see this paper Table A1 in Appendix:
https://doi.org/10.5194/gmd-16-3809-2023
